client/src/modules/list/components/ItemsContainer.jsx

```diff
@@ -9,8 +9,11 @@ import ItemsList from 'modules/list/components/Items';
 import { getCurrentUser } from 'modules/user/model/selectors';
 import { IntlPropType } from 'common/constants/propTypes';
 import './ItemsContainer.scss';
-import Filter from 'common/components/Filter';
+import Badge from 'common/components/Badge';
 import { CloseIcon } from 'assets/images/icons';
+import Filter from 'common/components/Filter';
+
+const badgeContext = 'items';
```

Comment:
how do you want to translate this?

Reply:
use https://github.com/formatjs/react-intl/blob/master/docs/Components.md#message-syntax inside the json locale file